### ISSUE TITLE
broken link

### ISSUE THREAD

**Issue by ebmarquez** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#668]
the link in the article is broken.
 "Getting started with Office 365 Connectors for Microsoft Teams"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f5eb6a35-32fa-6703-8ed2-0855e602b7d5
* Version Independent ID: 7890ca09-d505-35a9-071a-8662a4b19275
* Content: [Use Office 365 and custom connectors in Microsoft Teams](https://docs.microsoft.com/en-us/MicrosoftTeams/office-365-custom-connectors#feedback)
* Content Source: [Teams/Office-365-custom-connectors.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/Office-365-custom-connectors.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by AndreaBarr** [comment]
@ebmarquez Thank you for this feedback

We appricate you taking the time to report this link issue to us and making the docs better. I am getting this over to the Teams team for investigation. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
